.nvmrc

```diff
@@ -1 +1 @@
-12.18.4
\ No newline at end of file
+v16.15.1
```

Comment:
I had to update my node version for this to work, not entirely sure why I guess this file wasn't update on recent pushes?